scripts/OverlaySurvey.py

```diff
@@ -232,6 +237,39 @@ def run_survey(args):
         json.dump(merged_results, outfile)
 
 
+def simplify(args):
+    simplified_graph = []
+    graph = nx.read_graphml(args.graphmlInput)
+    bidirectional_adj = {}
+    for v in graph.nodes():
```

Comment:
Why do we need to do this? I think preferred peers should reflect _outbound_ peers of a node, so you need to keep the direction as-is.

Reply:
I added this part since I was thinking that the direction of an edge in the survey results tells us who sets the connection up initially, but once established all traffic flows both ways? I'm not 100% sure, though.

This also seems to make the simulation simpler. For instance, by making the graph bidirectional, we can ensure that all nodes have been connected to all the nodes it's supposed to be connected to by checking if `the # of nodes connected to node v` equals `the length of preferredPeers list`. If we don't make it bidirectional, we probably have to check something like `the preferredPeers list` ⊂ `the nodes connected to v`.

Reply:
nx can import graphs as undirected - so you don't need to do this manual tracking.
By using the right data structure, you will fix a bug: `peers` contains nodes that you deleted.

Reply:
ok clarification on my previous comment: @MonsieurNicolas  let me know that we can use both inbound and outbound peers (as opposed to just outbound) as preferred peers so that we can use the "preferred peers only" flag (the flag is useful to reduce potential churn when nodes need to reconnect, as it rejects any connections that are not preferred peers).